src/app/_services/api/export.service.ts

```diff
@@ -0,0 +1,20 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Serializer } from '../../_serializers/serializer';
+import { HelperService } from '../helper.service';
+import { BaseService } from './base.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ExportService extends BaseService<any> {
```

Comment:
ExportService is using jsut the export method right? It shouldn;t extend the BaseService

Reply:
Modified.